content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+date: 2019-11-13T11:23:04-06:00
+draft: false
+meta_desc: "Pulumi launches supports .NET Core languages for Infrastructure as Code"
+meta_image: "meta.png"
```

Comment:
I'm seeing this render a bit funny (overflowing the bounding box).

![Screen Shot 2019-11-13 at 1 26 22 PM](https://user-images.githubusercontent.com/223467/68805761-4df3e500-0619-11ea-8460-d9fb71503aa0.png)